### PR TITLE
Poetry2nixify

### PR DESCRIPTION
**Pull request by grahamc**
I couldn't actually get it to work because my libvirtd wasn't easy to set up. Maybe someone else can test it. 

testing steps:

    clone, check out the PR
    nix-shell
    poetry install
    poetry shell
    nixops create -d test tests/functional/....nix
    nixops deploy -d test

... make sure it works :P

### CODE REVIEW

**Comment by mmilata** [comment]
Followed the testing steps and encountered no error. Looks good!

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "nixops_virtd"
+version = "1.0"
+description = "NixOps plugin for virtd"
+authors = ["Amine Chikhaoui <amine.chikhaoui91@gmail.com>"]
+include = [ "nixops_libvirtd/nix/*.nix" ]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+nixops = {git = "https://github.com/adisbladis/nixops.git", rev = "poetry2nix-plugins"}
```
Comment (grahamc) [inline pyproject.toml:10]: Update after merging https://github.com/NixOS/nixops/pull/1256

Comment (grahamc) [reply pyproject.toml:10]: change this to nixos/nixops.git rev = "master"

**Comment by jhillyerd** [comment]
These steps are working for me.

**Comment by mgdm** [comment]
This works for me too

**Comment by jhillyerd** [comment]
@grahamc give these steps a try: https://gist.github.com/jhillyerd/38dd7fe1492b713c64686055c910dfa7

I forgot I had written https://github.com/nix-community/nixops-libvirtd/pull/15/files - but after experimenting last night I've determined the main thing missing is the storage pool for root, not a particular user, so it needs to be updated.  So I've flipped that to draft for now.

**Comment by typetetris** [comment]
How can I provide a nixops with the loaded plugin as an application? I don't want my colleagues to have to follow the above steps every time, they use nixops.

**Pull request merged**